.github/workflows/stale.yml

```diff
@@ -0,0 +1,40 @@
+name: Stale check
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 0 * * *'
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          close-issue-reason: not_planned
+          days-before-stale: 30
+          days-before-close: -1
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has
+            not had any activity in the past 30 days.
+
+            
+            If the opened issue is a bug, check if newer versions of the Agent have
+            fixed the issue. If it is not, please let us know is this issue is still
+            relevant otherwise, feel free to close it.
+
+            Thank you for your contributions!
+          stale-pr-message: >
+            This PR has been automatically marked as stale because it has not
+            had any activity in the past 30 days.
+
+            If the PR could not move forward because it has not enough reviewers, please ping
+            the maintainers in order to get more reviews. If you do not have enough
+            time to follow up on this PR, consider closing it and let us know.
+
+            Thank you for your contributions!
```

Comment:
I think these should be changed to be slightly more clear that no action is needed from the author of the issue/PR; something that indicates the label is being applied for tracking but their issue/PR will _not_ be automatically closed. 

In particular, I'm concerned that the calls to ping the maintainers has the same annoying effect on authors: they'll think they have to fight the stale bot to remove the label. I think if the messages make it clear that the label is applied for tracking purposes, then we might not need to include this call to action here, but we can still request that the author close their issue/Pr if it's no longer relevant.